db/scripts/init_tables.sql

```diff
@@ -23,3 +23,12 @@ CREATE TABLE IF NOT EXISTS domains
     data        TEXT NOT NULL,
     status      VARCHAR(255) NOT NULL
 );
+
+
+CREATE TABLE IF NOT EXISTS blacklisted_emails
+(
+    id          SERIAL PRIMARY KEY,
+    email       TEXT NOT NULL,
+    reason      TEXT NOT NULL,
+    timestamp   TIMESTAMP
+);
```

Comment:
This reminds me, we should probably talk about how we want to do database migrations. Should all of our scripts just be idempotent?

Reply:
It looks like there are a couple libraries for managing migrations but at the moment the schema seems simple enough that we could continue this way until we need one, yeah.

Reply:
SG. Just to document our offline convo, we could:
 - Version our database and build tooling to automatically select which migrations are needed to upgrade it.
 - Do the thing we're doing now, which is to write idempotent scripts that run every time the container restarts.

And we decided to continue going with the latter until we have a use case that is complex enough to need the former.